client/src/PaginatorControls.jsx

```diff
@@ -1,92 +1,154 @@
-var React = require('react');
-var MagicLabeler = require('./MagicLabeler');
-var PaginatorControlGroup = require('./PaginatorControlGroup');
-var ZoomToggleButton = require('./ZoomToggleButton');
+const React = require("react");
+const PropTypes = require("prop-types");
 
-class PaginatorControls extends React.Component{
-    approveCurrentPageLabel = () => {
+const JobPaginator = require("./JobPaginator");
+const MagicLabeler = require("./MagicLabeler");
+const PaginatorControlGroup = require("./PaginatorControlGroup");
+const ZoomToggleButton = require("./ZoomToggleButton");
+
+class PaginatorControls extends React.Component {
+    constructor(props) {
+        super(props);
+        // TODO: Why do we have to do this now?
```

Comment:
It’s because of the switch away from fat arrow notation. It was throwing an error because it’s an ES7 feature[1]. I should be able to bump the eslint parser version setting and restore order to the galaxy.

[1] https://blog.logrocket.com/a-guide-to-react-onclick-event-handlers-d411943b14dd/

Reply:
Excellent!